examples/az_cli.py

```diff
@@ -74,3 +75,49 @@ def CreateDiskCopy(args: 'argparse.Namespace') -> None:
                                        src_profile=args.src_profile,
                                        dst_profile=args.dst_profile)
   logger.info('Done! Disk {0:s} successfully created.'.format(disk_copy.name))
+
+
+def ListMetrics(args: 'argparse.Namespace') -> None:
+  """List Azure Monitoring metrics for a resource.
+
+  Args:
+    args (argparse.Namespace): Arguments from ArgumentParser.
+  """
+  az_account = account.AZAccount(args.default_resource_group_name)
+  az_monitoring = monitoring.AZMonitoring(az_account)
+  metrics = az_monitoring.ListAvailableMetricsForResource(args.resource_id)
+  for metric in metrics:
+    logger.info('Available metric: {0:s}'.format(metric))
+
+
+def QueryMetrics(args: 'argparse.Namespace') -> None:
+  """Query Azure Monitoring metrics for a resource.
+
+  Args:
+    args (argparse.Namespace): Arguments from ArgumentParser.
+
+  Raises:
+    RuntimeError: If from_date or to_date could not be parsed.
+  """
+  az_account = account.AZAccount(args.default_resource_group_name)
+  az_monitoring = monitoring.AZMonitoring(az_account)
+  from_date, to_date = args.from_date, args.to_date
+  if from_date and to_date:
+    try:
+      from_date = datetime.strptime(from_date, '%Y-%m-%dT%H:%M:%SZ')
+      to_date = datetime.strptime(to_date, '%Y-%m-%dT%H:%M:%SZ')
+    except ValueError as exception:
+      raise RuntimeError('Cannot parse date: {0:s}'.format(str(exception)))
+  metrics = az_monitoring.GetMetricsForResource(
+      args.resource_id,
+      metrics=args.metrics,
+      from_date=from_date,
+      to_date=to_date,
+      interval=args.interval,
+      aggregation=args.aggregation or 'Total',
+      qfilter=args.qfilter)
+
+  for metric in metrics:
+    logger.info('Metric: {0:s}'.format(metric))
+    for value in metrics[metric]:
+      logger.info('Value: {0:s}'.format(value))
```

Comment:
```suggestion
      logger.info(' Value: {0:s}'.format(value))
```
So that things are visually indented?